src/taipy/gui/_renderers/__init__.py

```diff
@@ -32,27 +23,28 @@
 
 class _Renderer(Page, ABC):
     def __init__(self, **kwargs) -> None:
-        """Initialize a new _Renderer with the indicated content.
+        """NOT DOCUMENTED
+        Initialize a new _Renderer with the indicated content.
 
         Arguments:
             content (str): The text content or the path to the file holding the text to be transformed.
 
         If *content* is a path to a readable file, the file is read entirely as the text template.
         """
         super().__init__(**kwargs)
-        content: t.Optional[t.Union[str, ElementApi]] = kwargs.get("content", None)
+        content: t.Optional[t.Union[str, "_Element"]] = kwargs.get("content", None)
         if content is None:
             raise ValueError("'content' argument is missing for class '_Renderer'")
         self._content = ""
-        self._base_element: t.Optional[ElementApi] = None
+        self._base_element: t.Optional["_Element"] = None
         self._filepath = ""
         if isinstance(content, str):
             self.__process_content(content)
-        elif isinstance(content, ElementApi):
+        elif any(filter(lambda c: c.__name__.endswith("_Element"), type(content).__mro__)):
```

Comment:
a bit scary

Reply:
Indeed
Let's fix that if you want.

Reply:
So far, we failed...
@dinhlongviolin1, any idea?